files/bxslider/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "bxslider-4",
+  "repo": "stevenwanderski/bxslider-4",
+  "files": {
+    "include": ["jquery.bxslider.min.js", "jquery.bxslider.js", "./images/*", "./plugins/*"]
```

Comment:
I dont think that `jquery.bxslider.js` is needed, the min file is enough right?
Also what about the css file? Its not needed in production?